spec/samples/three_clean_files/clean_two.rb

```diff
@@ -1,5 +1,9 @@
 # clean class for testing purposes
 class Clean
+  def initialize(sub)
+    @sub = sub
+  end
+
```

Comment:
Yep, that's what I meant with: https://github.com/troessner/reek/issues/978
We really need to address this before writing further detectors (this one is ok, talking about the ones after that).